packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.scss

```diff
@@ -24,6 +24,8 @@
       height: auto;
       width: 100%;
       @include text-align(left);
+      padding: 4px 20px 6px;
```

Comment:
Nit: It's often confusing when there are three values, `padding: 4px 20px 6px 20px` can prevent future problems.